spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -590,9 +590,38 @@
     expect(filter.to_ruby).to eq('<value ref=vm, type=integer>/virtual/used_disk_storage</value> >= 1048576000')
   end
 
+  it 'should test .is_numeric and .is_integer' do
```

Comment:
This looks great. What do you think about separating these into blocks for each method, and a separate spec for each behavior you're testing? I've noticed we do that a lot in this file, but it's not necessarily a good pattern to follow